configure.ac

```diff
@@ -585,6 +585,8 @@ then
     test "$enable_trusted_ca" = "" && enable_trusted_ca=yes
     test "$enable_session_ticket" = "" && enable_session_ticket=yes
 
+    test "$enable_earlydata" = "" && enable_earlydata=yes
```

Comment:
I am okay with early data on for `--enable-all`, but I don't think we should have it on for `--enable-distro`, which is used for Debian. @SparkiDev thoughts?

Reply:
I agree.
EarlyData is one of those features where customers could use it without understanding the security limitations.
Having said that, they need to use specific APIs to send and get the data.
I would prefer to have it manually turned on in the distribution.

Reply:
Understood. I disable early data for `--enable-distro`